src/infrastructure/persistence/PsqlTestRepository.ts

```diff
@@ -50,6 +50,7 @@ export class PsqlTestRepository implements TestRepository {
       new TestId(linkRow.id),
       new UserId(linkRow.userId),
       new TestTypeId(linkRow.testTypeId),
+      undefined,
```

Comment:
Note, this looks weird!  At this point, the persistence for results is not implemented (comes in follow on PR).  I rearranged the order of the constructor params, so needed this temporarily.

Reply:
Yep I saw this in the following PR